pkg/apis/core/types.go

```diff
@@ -498,6 +498,11 @@ type PersistentVolumeClaimStatus struct {
 	Capacity ResourceList
 	// +optional
 	Conditions []PersistentVolumeClaimCondition
+	// AllocatedResources stores resources that were allocated for the PVC.
+	// This field is used for tracking of pvc size allocation towards which
+	// volume expansion controller might be working on.
+	// +optional
```

Comment:
Can you add the following to this comments here?
`Reduction in AllocatedResources is only done by volume expansion controller after verifying volume size following a completed volume expansion.`

Reply:
I have incorporated your and @msau42 wording below. let me know if this looks good.